kge/model/embedder/lookup_embedder.py

```diff
@@ -29,9 +28,18 @@ def __init__(
             self.dim = round_to_points(round_embedder_dim_to, self.dim)
 
         # setup embedder
-        self._embeddings = torch.nn.Embedding(
-            self.vocab_size, self.dim, sparse=self.sparse
-        )
+        if self.normalize_p > 0:
+            self._embeddings = torch.nn.Embedding(
+                self.vocab_size,
+                self.dim,
+                sparse=self.sparse,
+                norm_type=self.normalize_p,
+                max_norm=1.0,
+            )
```

Comment:
This is not correct. We also want vectors of norm < 1 to be normalized.

Reply:
Yes I wanted to discuss this properly this time: The only code example for embedding normalization in TransE that I found was doing it like this, i.e. only normalize when norm > 1. We were not sure why and ad hoc decided to always normalize. So the question is now what we want to do: do we replicate what somebody else did for TransE or just always normalize.

Reply:
The TransE paper proposes to normalize. If we wanted max_norm (we do not for now, I think), it should be a separate option.

Reply:
From Bordes' implementation seems that he always normalizes as well:

https://github.com/glorotxa/SME/blob/a1aa9fdb790fd356a17cd13d2b7b75feba2d8c82/FB15k_exp.py#L267

https://github.com/glorotxa/SME/blob/a1aa9fdb790fd356a17cd13d2b7b75feba2d8c82/model.py#L277

(I vaguely remembering looking at this code already, maybe this why we chose to always normalize back then).